etc/DependencyInstaller.sh

```diff
@@ -355,15 +418,45 @@ case "${platform}" in
 esac
 
 case "${os}" in
-    "CentOS Linux" )
+    "CentOS Linux" | "Red Hat Enterprise Linux Server" | "AlmaLinux" | "Rocky Linux" | "Red Hat Enterprise Linux" )
```

Comment:
I think `"Red Hat Enterprise Linux"*` would cover both `Red Hat Enterprise Linux` and `Red Hat Enterprise Linux Server`, no?